app/lib/tasklist_content.rb

```diff
@@ -0,0 +1,13 @@
+class TasklistContent
+  def self.learn_to_drive_config
```

Comment:
Because we modify this content in place to show which link is active, we need to instantiate a new one for each request.  See https://github.com/alphagov/frontend/blob/be554f44a57e65f227d48229ea0193e1c315ff4c/lib/tasklist_content.rb .  Otherwise we get duplicate active sections and multiple tennis balls because we don't unset the active items.  (Screenshot below)

![screen shot 2017-11-03 at 08 31 34](https://user-images.githubusercontent.com/773037/32365580-322f006e-c072-11e7-9142-008229efe5f3.png)